Docker.md

```diff
@@ -92,11 +92,9 @@ In case you need to inspect the database, you need a second terminal
 and run:
 
 ```
-docker-compose run database mysql -hdatabase -ulinuxfr_rails -p linuxfr_rails
+docker-compose run database mysql -hdatabase -ulinuxfr_rails -plinuxfr_rails linuxfr_rails
 ```
 
-By default, the requested password is the same as the username.
-
```

Comment:
The goal of the documentation is to be explicit, I'd prefer to keep this sentence to express this is configurable. Maybe we can add another sentence to point where this can be made (either within the deployment/default.env or docker-compose.yaml file).